rest-service/manager_rest/test/storage/base_test.py

```diff
@@ -0,0 +1,128 @@
+#########
+# Copyright (c) 2013 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+import os
+import json
+
+import unittest
+import tempfile
+
+from nose.plugins.attrib import attr
+
+from manager_rest import (
+    config,
+    constants,
+    utils
+)
+from manager_rest.storage import (
+    models,
+    get_storage_manager,
+)
+
+from manager_rest.test.security_utils import get_admin_user
+
+LATEST_API_VERSION = 3  # to be used by max_client_version test attribute
+
+
+@attr(client_min_version=1, client_max_version=LATEST_API_VERSION)
+class ModelsIntegrationTestsBase(unittest.TestCase):
+    def setUp(self):
+        rest_service_log, tmp_conf_file = \
+            self._create_temp_files_and_folders()
+        self.server = self._set_config_path_and_get_server_module(
+            rest_service_log, tmp_conf_file)
+        self.server_configuration = self._create_config_and_reset_app(
+            rest_service_log)
+        self._set_flask_app_context()
```

Comment:
Create a test base class in tests/base_test.py that doesn't use clients and stuff.
BallerTestCase